### PR TITLE
feat: VIC-1825 create API that enables asker's email update

### PR DESCRIPTION
**Pull request by idrissnaji**
Related PR https://github.com/Onlineberatung/onlineBeratung-userService/pull/551

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -25,6 +26,8 @@ keycloakService.consultant.role=consultant
 keycloakService.technical.username=
 keycloakService.technical.password=
 
+identity.email-dummy-suffix=@beratungcaritas.de
```
Comment (idrissnaji) [inline src/main/resources/application.properties:29]: FYI: we use the same key value in userService

File: src/main/java/com/vi/appointmentservice/repository/CalcomRepository.java [hunk 2]
```diff
@@ -79,4 +79,11 @@ public void deleteAttendeeWithoutBooking() {
     calcomDBNamedParamterTemplate.update(QUERY, new MapSqlParameterSource());
   }
 
+  public void updateAttendeeEmail(final Long bookingId, final String email) {
```
Comment (adnanalicic) [inline src/main/java/com/vi/appointmentservice/repository/CalcomRepository.java:82]: would probably be more performant in case of "where in".

**Review by adnanalicic — APPROVED** [review]

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -14,6 +14,7 @@ calcom.database.url=${calcom_database_url}
 calcom.database.username=${calcom_database_username}
 calcom.database.password=${calcom_database_password}
 calcom.database.driverClass=org.postgresql.Driver
+calcom.email.trash=${calcom_email_trash}
```
Comment (adnanalicic) [inline src/main/resources/application.properties:17]: please add the env variable to k8s_digi_config project

File: src/main/java/com/vi/appointmentservice/repository/CalcomRepository.java (hunk 2 above)
Comment (idrissnaji) [reply src/main/java/com/vi/appointmentservice/repository/CalcomRepository.java]: done

**Pull request merged**